BootstrapKit/ModerationPolicy.md

```diff
@@ -0,0 +1,14 @@
+## Working Group Moderation Policy Template
+
+Every Working Group must have a Moderation Policy. By default,
```

Comment:
Yes, this is ideal. Keep in mind tho that this language is a departure from our prior policies (which are kind of a failure in this regard so +1 to the change).